cmd/processor/app/processor.go

```diff
@@ -649,3 +657,49 @@ func (p *Processor) setWorkersStatus(triggerInstance trigger.Trigger, status sta
 
 	return nil
 }
+
+// creates a signal handler, so on signal processor gracefully terminated
```

Comment:
A Go convention - add the function name at the beginning of the function docstring